app/controllers/admin/configs_controller.rb

```diff
@@ -208,6 +209,17 @@ def clean_up_params
       config[:credit_prices_in_cents]&.transform_values!(&:to_i)
     end
 
+    def toggle_email_password_authentication
```

Comment:
This method updates the email-registration and email-login fields; I also update the invite-only field according to the [design specs](https://user-images.githubusercontent.com/816402/96673429-58375c80-131b-11eb-848a-0ccacec0e0cf.jpg), which indicate that turning email-auth off should also disable `invite-only` mode.